include/ignition/gui/Conversions.hh

```diff
@@ -91,12 +91,14 @@ namespace ignition
     /// \param[in] _t The time to convert
     /// \return An ignition::common::Time object
     IGNITION_GUI_VISIBLE
+    IGN_DEPRECATED(4)
     common::Time convert(const msgs::Time &_t);
 
     /// \brief Convert an ignition::common::Time to an ignition::msgs::Time
     /// \param[in] _t The time to convert
     /// \return An ignition::msgs::Time object
     IGNITION_GUI_VISIBLE
+    IGN_DEPRECATED(4)
     msgs::Time convert(const common::Time &_t);
```

Comment:
How about adding new functions that use `chrono` instead of `ignition::common`? Whoever is using these functions now will need an alternative.

Reply:
Oh now it's possible to use the ones from `ign-msgs` added in https://github.com/ignitionrobotics/ign-msgs/pull/81, I'll just mention that in the migration guide.